tasks/certificate-custom.yml

```
@@ -0,0 +1,18 @@
+---
+- name: Deploy custom private key
+  become: true
+  copy:
+    src: '{{ bbb_own_key }}'
+    dest: '{{ bbb_ssl_key }}'
+    owner: "root"
+    group: "{{'ssl-cert' if ansible_os_family == 'Debian' else 'root' }}"
+    mode: '0440'
+  notify: reload nginx
+
+- name: Deploy custom certificate
+  become: true
+  copy:
+    src: '{{ bbb_own_cert }}'
+    dest: '{{ bbb_ssl_cert }}'
+    mode: '0644'
+  notify: reload nginx
```

```suggestion
  notify: reload nginx

```
Add newline in end of file